.github/workflows/deploy-migration-test.yml

```
@@ -0,0 +1,201 @@
+
+jobs:
+  build-lambda-handler:
+    name: Build Lambda Handler
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)